.github/workflows/prices-api.yaml

```
@@ -0,0 +1,27 @@
+name: Prices API
+description: Builds and publishes the Prices API package to NPM
+
+on:
+  push:
+    branches:
+      - main
```

We dont need a new version every push. This would probably fail because of duplicated versions
[on: tags](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs) is better in this case I think
```suggestion
    tags:
      - prices-api-v**
```

---

opted for the more specific `prices-api/[0-9]+.[0-9]+.[0-9]+` tag